### ISSUE TITLE
Im blind plz dark

### ISSUE THREAD

**Issue by jacobtread** [Fallstop/HCNoticeReader#1]
Plz add dark theme

**Comment by jacobtread** [comment]
Brr add the following to their respective selectors for simple darkmode
```css
main.svelte-1c1p02y .PrimaryContent.svelte-1c1p02y {
      background-color: #444;
}

body {
   background: #333;
    color: #999;
}
```

**Comment by Fallstop** [comment]
Forgot I was also blind, adding dark mode switch

**Comment by Fallstop** [comment]
Fixed in e0ccc09b0d15437a1f7e4648447710112f3ee7e4.
